src/Html.php

```diff
@@ -5577,6 +5577,8 @@ class='form-control'
             maxFileSize: {$max_file_size},
             maxChunkSize: {$max_chunk_size},
             add: function (e, data) {
+               // disable submit button during upload
+               $(this).closest('form').find('button:submit').prop('disabled', 'disabled');
```

Comment:
1. Using `:submit` will also disable `<input type="submit" />` elements (can be used is specific forms).
2. `$().prop()` value parameter should be a boolean, according to documentation.
```suggestion
               $(this).closest('form').find(':submit').prop('disabled', true);
```